src/beta/resources/end_shipper.js

```diff
@@ -0,0 +1,15 @@
+import base from '../../resources/base';
+import { baseAddress as baseAddressPropTypes } from '../../resources/base_address';
+
+export const propTypes = Object.assign({}, baseAddressPropTypes);
```

Comment:
We shouldn't need this line

Reply:
I believe we need it for this line: https://github.com/EasyPost/easypost-node/blob/end_shipper/src/beta/easypost.js#L2

`{ propTypes as endShipperPropTypes }`